utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -49,17 +49,28 @@ def build(self, sharedlib_only=False, model_only=False):
                          sharedlib_only=sharedlib_only, model_only=model_only)
 
     def run(self):
+        return self._run()
+
+    def _run(self, suffix="base"):
         rc, out, err = run_cmd("./case.run --caseroot %s" % self._caseroot, ok_to_fail=True)
-        if rc == 0 and self.coupler_log_indicates_run_complete():
```

Comment:
In a multirun testcase, self._runstatus could have had a FAIL masked by a later PASS